Library/Formula/cowpatty.rb

```diff
@@ -11,7 +11,12 @@ class Cowpatty < Formula
     sha256 "35fba0f92c5e8fb0710453d0c2c5fe5e9c64857dd53b219977871340b22c4942"
   end
 
+  def patches
+    [ "http://proton.cygnusx-1.org/~edgan/cowpatty/cowpatty-4.6-fixup16.patch" ]
```

Comment:
Thins patch probably never will be in upstream because this project is very old and without a maintainer. Latest version is from July 03 2009 and homepages are not working. However this patch is absolutely neccessary if you want to safe half time of wpa cracking with the use of stdout from pyrit or other similar software and it is recommended way how to get cowpatty on Debian, Ubuntu and other distros. Unfortunatelly only references to that are on several blogs..